project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 resolvers ++= DefaultOptions.resolvers(snapshot = true)
 
-addSbtPlugin("com.typesafe.play" % "interplay" % "1.1.2")
-addSbtPlugin("com.typesafe.play" % "play-docs-sbt-plugin" % sys.props.getOrElse("play.version", "2.5.9"))
+addSbtPlugin("com.typesafe.play" % "interplay" % "1.3.4")
```

Comment:
@wsargent for whatever reason, this plugin update is breaking the test execution. Updating only to 1.2.0 makes everything fine.